kytos/core/interface.py

```diff
@@ -68,6 +73,31 @@ def __repr__(self):
         return f"TAG({self.tag_type!r}, {self.value!r})"
 
 
+# pylint: disable=super-init-not-called
+class TAGRange(TAG):
+    """Class that represents an User-to-Network Interface with
+     a tag value as a list."""
+
+    # pylint: disable=dangerous-default-value
+    def __init__(
+        self,
+        tag_type: str,
+        value: list[list[int]],
+        mask_list: list[Union[str, int]] = []
```

Comment:
@Alopalao, in python land, trying to set a mutable list as an argument can result in certain issues https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument, if you could revert it back as `Optional[list[Union[str, int]]` = None, and then if it's None setting an empty list inside the `__init__` method that'd be the safest route.